lib/resolvers/project.js

```diff
@@ -622,5 +622,19 @@ module.exports = ({ models }) => async ({ action, data, id, meta = {}, changedBy
     return Project.query(transaction).patchAndFetchById(id, { refusedDate });
   }
 
+  if (action === 'suspend') {
+    const project = await Project.query(transaction).findById(id);
+
+    if (project.status !== 'active') {
+      throw new Error('Can only suspend active projects');
+    }
```

Comment:
Is there a reason this check is only there for PPLs?

Reply:
Not really, other than we're doing similar checks for revoke/refuse only in projects.